### PR TITLE
Added sanitize fields as objects

### PR DESCRIPTION
**Pull request by ferdie11**

### CODE REVIEW

File: src/utils/sanitizer.js [hunk 1]
```diff
@@ -1,5 +1,22 @@
 import filters from 'xss-filters';
+import _ from 'lodash';
 
 export default {
-    sanitize: filters.inHTMLData
+    sanitize: function(attribute) {
+        if (_.isObject(attribute) && !_.isNull(attribute)) {
+            if (_.isArray(attribute)) {
+                attribute.forEach(function(index) {
```
Comment (joshystuart) [inline src/utils/sanitizer.js]: Just curious why you use the native for loop here, and the `lodash` one below?

File: test/integration/controllers/patch.js [hunk 2]
```diff
@@ -154,6 +154,37 @@ describe('Integration Tests', function() {
...
 
+            it('should sanitize nested fields', function(done) {
+                const id = admins[0]._id;
+                const updates = {
+                    data: {
+                        id: id,
+                        attributes: {
+                            'first-name': '<script>Watermelon</script>',
+                            'last-name': '<script>alert("xss")</script>',
+                            address: {
```
Comment (joshystuart) [inline test/integration/controllers/patch.js:165]: Should there also be a test where there is an array of `addresses` ?

File: src/utils/sanitizer.js (hunk 1 above)
Comment (ferdie11) [inline src/utils/sanitizer.js]: Right. I missed that. Fixing.

File: test/integration/controllers/patch.js (hunk 2 above)
Comment (ferdie11) [inline test/integration/controllers/patch.js:165]: Gotcha. Adding it now.

**Pull request merged**